fix: Part fix #264. Remove end space. Use consistent monospace

ltfschoen opened the pull request:
Addresses _"2. Param rendering needs to remove extra space at end"_ of the to top-of-mind issues mentioned at https://github.com/polkadot-js/apps/issues/264#issuecomment-415955065

* Trims the address of preceding and trailing whitespace
* Shows CopyButton on the right of each address
* Uses Courier/monospace font family so the CopyButton's are vertically aligned when multiple addresses are returned from a query

Screenshot:

<img width="812" alt="screen shot 2018-09-10 at 8 24 03 am" src="https://user-images.githubusercontent.com/6226175/45280140-80fd8e80-b4d3-11e8-8ae8-55398b030b30.png">

ltfschoen commented:
@jacogr I've retained the monospace font, removed the CopyButton, and added an empty div so double clicking the address only selects the address

jacogr commented on packages/ui-app/src/Params/Params.css:
```diff
@@ -2,6 +2,10 @@
 /* of the ISC license. See the LICENSE file for details. */
 
+.ui--Param-Address {
+  font-family: 'Courier New', Courier, monospace;
```
Just make this `monospace` consistent with what we have in the rest of the UI, e.g. https://github.com/polkadot-js/apps/blob/master/packages/ui-app/src/styles/components.css#L54

ltfschoen commented:
@jacogr I've added a key prop the empty div (that allows the user to double click and select the address) so it doesn't produce missing key error. I've made monospace the preferred font and added Courier fonts as fallback incase the user doesn't support it. I've added the PostCSS Mixins dependency so we can consistently use the same font family or other traits for different hashes that we are displaying across the UI

chevdor commented:
Feel free to close #269 once this PR is merged.

jacogr commented on packages/ui-app/src/styles/partials/_variables.css:
```diff
@@ -24,3 +28,6 @@ $button-negative-color-hover: $middle-purple;
 $progress-bar-color: $dark-sky-blue;
+
+/* fallback font families included */
+$hash-font-family: monospace, 'Courier New', Courier;
```
Actually `monospace` is whatever the browser decides it to be, so actually no need for a fallback, it is the built-in that is always there. https://developer.mozilla.org/en-US/docs/Web/CSS/font-family

So all these mixin changes can be reverted. Just `monospace` is fine and works by default.

ltfschoen commented:
@jacogr I've reverted the mixins and just used monospace now
